anchore_engine/services/policy_engine/api/controllers/feeds.py

```diff
@@ -182,14 +197,18 @@ def toggle_group_enabled(feed, group, enabled):
 
 @authorizer.requires_account(with_types=INTERNAL_SERVICE_ALLOWED)
 def delete_feed(feed):
-    if feed == GRYPE_DB_FEED_NAME:
+    provider = get_vulnerabilities_provider()
+    internal_feed_name = provider.display_mapper.get_internal_name(feed)
+    if isinstance(provider, GrypeProvider) and internal_feed_name == GRYPE_DB_FEED_NAME:
         raise HTTPNotImplementedError(
-            message="Deleting the grypedb feed is not yet supported.",
+            message="Deleting the {} feed with the grype vulnerability provider enabled is not yet supported.".format(
```

Comment:
Can you use the safer string format option, e.g.: [https://github.com/anchore/anchore-engine/blob/6ca1795bf20114a173ab348d8184c2e647c987d1/anchore_engine/clients/grype_wrapper.py#L309](https://github.com/anchore/anchore-engine/blob/6ca1795bf20114a173ab348d8184c2e647c987d1/anchore_engine/clients/grype_wrapper.py#L309)

Reply:
switched to f-strings